### ISSUE TITLE
aws_customer_gateway parse/syntax error

### ISSUE THREAD

**Issue by kkrusedp** [hashicorp/terraform#3241]
```
$ cat vpn.tf
resource "aws_vpn_gateway" "vpn_gateway" {
      vpc_id = "${aws_vpc.vpc.id}"
}

resource "aws_customer_gateway" "customer_gateway" {
      bgp_asn = "65000"
      ip_address = "1.2.3.4"
      type = ipsec.1
}
resource "aws_vpn_connection" "main" {
      vpn_gateway_id = "${aws_vpn_gateway.vpn_gateway.id}"
      customer_gateway_id = "${aws_customer_gateway.customer_gateway.id}"
      type = "ipsec.1"
      static_routes_only = true
}

```

 I copied this from https://terraform.io/docs/providers/aws/r/vpn_connection.html

This was copied from the docs so I suspect some bug here.  When you run returns the following:

```
$ terraform plan
Error loading config: Error parsing /path/to/vpn.tf: Line 9, column -1: syntax error
```

**Comment by apparentlymart** [comment]
I think the problem is that the ``aws_customer_gateway.customer_gateway`` resource needs its ``type`` to be quoted.

You're right that the bug is in [the documentation](https://github.com/hashicorp/terraform/blob/master/website/source/docs/providers/aws/r/vpn_connection.html.markdown), but as a workaround for now I think you can just put quotes around ``ipsec.1`` there.

**Comment by catsby** [comment]
Thanks! I fixed the documentation in b7cbd08 , it will go out in the next release
